Applications/WebPortal/WebApp/library/source/pages/account_page/comment_box.tsx

```diff
@@ -5,13 +5,20 @@ interface Properties {
 
   /** The text to display in the comment box. */
   comment: string;
+
+  /** The callback function that updates the comment. */
+  onInput(value: string): void;
 }
 
 /** Displays the comment box. */
 export class CommentBox extends React.Component<Properties> {
   public render(): JSX.Element {
     return <textarea className={css(CommentBox.STYLE.submissionBox)}
-      value={this.props.comment} placeholder='Leave comment here…'/>;
+      value={this.props.comment}
+      onChange={(event: any) => {
```

Comment:
Find a way to do this without the ```any```. Should be something like ```React.InputEvent```

Reply:
From everything I've read about forms in react that doesn't seem to be possible.